lib/app/admin/data-reset/create.js

```diff
@@ -7,25 +6,45 @@ var DEFAULT_LIST_RELATIONS = require('./default_list_relations');
 var DEFAULT_ITEM_LIST = require('./default_item_type_list');
 
 
+function getBuildDataFunction(mediator) {
+  return function buildData() {
+    var deferred = q.defer();
+    var relationsMap = [];
+    console.log('>>>>>Start create');
+    async.eachSeries(DEFAULT_ITEM_LIST, function(itemType, eachTypeCb) {
```

Comment:
Mixing async callbacks and promises is usually a code smell, and certainly not something I'd put in a reference implementation if I could help it. It's not something I would want people to copy/paste.

The Q library doesn't provide any sort of map series function, but Bluebird definitely does, and would clean this code up a lot. It also has a few nice utility methods like `fromCallback` and `asCallback` for converting between old-style callbacks and promises.

At the very least, I would still get away from using `defer()`, and move to using `Q(function(resolve, reject) { ... })`

Also, in the callback for `createItemsForType()`, you're not handling the error at all.